app/models/solarcycle.rb

```diff
@@ -1,4 +1,6 @@
 class Solarcycle < ApplicationRecord
   self.inheritance_column = :_type_disabled
   belongs_to :location
+
+  scope :after_beginning_of_yesterday, -> { where('created_at < ?', 1.day.ago.beginning_of_day) }
```

Comment:
Shouldn't this be `where('created_at > ?', 1.day.ago.beginning_of_day)` ?

Reply:
Doh! You're write. If only I had written unit tests ;)